controllers/release/adapter.go

```diff
@@ -751,8 +912,10 @@ func (a *adapter) validatePipelineSource() *controller.ValidationResult {
 		return a.validationError(err)
 	}
 
-	if releasePlan.Spec.Pipeline != nil {
-		pipelineRef = releasePlan.Spec.Pipeline.PipelineRef
+	if releasePlan.Spec.TenantPipeline != nil {
+		pipelineRef = releasePlan.Spec.TenantPipeline.PipelineRef
+	} else if releasePlan.Spec.FinalPipeline != nil { // TODO: Refactor to remove the else if
```

Comment:
I don't think we want this else if, maybe we should talk about it in standup. Honestly, this function should probably make sure each of the three pipelineRefs (or whichever are defined) is not cluster scoped. Currently, it does not check the managed one if a tenant pipeline is defined, but it should IMO

Reply:
I changed it like you suggested in the call